core/src/i18n/en.yaml

```
@@ -1002,6 +1002,28 @@ git-repositories:
     none: Authentication not required.
     secret: The name of the Secret with credentials used by the Function Controller to authenticate to the Git repository in order to fetch the Function's source code and dependencies.
     url: URL must be a valid Git repository address that starts with the "http(s)", "git", or "ssh" prefix.
+ingresses:
+  description: <0>Ingress</0> describes the routing rules for traffic to services within a cluster. The routing is managed by the Ingress resource.
+  labels:
+    apiGroup: ApiGroup
```

```suggestion
    apiGroup: APIGroup
```
`API` is an acronym for `Application Programming Interface`, unless there's a reason why it's spelt this way here that I'm not aware of.